src/style.css

```diff
@@ -0,0 +1,55 @@
+body {
+  margin: 0;
+}
+
+.nav-bar {
+  display: flex;
+  height: 60px;
+  justify-content: space-between;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  padding-right: 50px;
+}
+
+.logo-img {
+  margin-top: 10px;
+  margin-left: 50px;
```

Comment:
FYI: There is a short property: `margin: top right bottom left;`
`margin: 10px 0 0 50px;`